routing/payment_lifecycle.go

```diff
@@ -803,8 +803,8 @@ func (p *paymentLifecycle) handleSwitchErr(attempt *channeldb.HTLCAttempt,
 	// case we can safely send a new payment attempt, and wait for its
 	// result to be available.
 	if errors.Is(sendErr, htlcswitch.ErrPaymentIDNotFound) {
```

Comment:
I think we should differentiate between the payment not found in the network resutts or the attempts, there are imo two different things:

https://github.com/lightningnetwork/lnd/blob/66940024acb3f96c7111132875c5f749e9ba3ca5/htlcswitch/payment_result.go#L235-L255

Reply:
wdyt ?